### DISCUSSION TITLE
How to make a production build that includes the dependencies?

### DISCUSSION THREAD

**Discussion by silviubogan** [jaredpalmer/razzle#1649] [Q&A] [answered]
I thought that I can move the build directory anywhere, on any server, and it would work, but it doesn't. It tells me that I have to install express, but this means creating a package.json only for deployment or copying a lot of useless files to the server. How can I make a production build that includes the dependencies? If I run locally `NODE_ENV=production node build/server.js` it runs well, I think it works because I have `node_modules` directory in the  parent directory.

If I run on a server `~/razzle-build# NODE_ENV=production node server.js` I get:

![image](https://user-images.githubusercontent.com/198924/124262365-9e93bb00-db3a-11eb-822b-ebe6b4b0ec2a.png)

Please help.

Thanks.

**Comment by fivethreeo** [comment, marked as answer]
https://razzlejs.org/docs/customization#options

buildType iso-serverless

**Reply by max-l** [reply]
@fivethreeo  could you give me an example of how the options are modified, I tried this a few ways (like below), without success

```
module.exports = {
    modifyWebpackConfig({
      env: {
        target, // the target 'node' or 'web'
        dev, // is this a development build? true or false
      },
      webpackConfig, // the created webpack config
      webpackObject, // the imported webpack node module
      options: {
        razzleOptions: { buildType: 'iso-serverless'}, // the modified options passed to Razzle in the `options` key in `razzle.config.js` (options: { key: 'value'})
        webpackOptions, // the modified options that will be used to configure webpack/ webpack loaders and plugins
      },
      paths, // the modified paths that will be used by Razzle.
    }) {
```

**Comment by fivethreeo** [comment]
https://github.com/jaredpalmer/razzle/blob/master/examples/basic-spa/razzle.config.js
